roles/servicetelemetry/tasks/component_snmp_traps.yml

```diff
@@ -0,0 +1,28 @@
+- name: Lookup template - Deployment
+  debug:
+    msg: "{{ lookup('template', './manifest_snmp_traps_deploy.j2') | from_yaml }}"
+
+- name: Set default snmp traps manifest
+  set_fact:
+    snmp_traps_manifest_deploy: "{{ lookup('template', './manifest_snmp_traps_deploy.j2') | from_yaml }}"
+  when: snmp_traps_manifest is not defined
+
+- name: Create an instance of snmp webhook
+  k8s:
+    definition:
+      '{{ snmp_traps_manifest_deploy }}'
+
+
+- name: Lookup template - Service
+  debug:
+    msg: "{{ lookup('template', './manifest_snmp_traps_service.j2') | from_yaml }}"
+
+- name: Set default snmp traps manifest - Service
+  set_fact:
+    snmp_traps_manifest_service: "{{ lookup('template', './manifest_snmp_traps_service.j2') | from_yaml }}"
+  when: snmp_traps_manifest is not defined
```

Comment:
If someone overrides the `snmp_traps_manifest` then you won't get a Service created.

Reply:
True. Is that an issue?

Reply:
I would think so. If you override the deployment manifest, you won't get a corresponding service, and there is no way to pass that to STO as this code stands, which would not be intuitive.